modules/web-console/frontend/app/modules/agent/AgentManager.service.ts

```diff
@@ -315,7 +321,8 @@ export default class AgentManager {
 
     saveToStorage(cluster = this.connectionSbj.getValue().cluster) {
         try {
-            localStorage.cluster = JSON.stringify(cluster);
+            if (cluster)
```

Comment:
Why we should try to restore cluster which gone

Reply:
Cluster will be restored only when it exists in list of connected cluster. In other cases first cluster will be selected.